crypto/rand/rand_lib.c

```diff
@@ -228,11 +228,9 @@ static uint64_t get_timer_bits(void)
     }
 #else
 
-# if defined(_POSIX_C_SOURCE) \
+# if (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 199309L) \
      && defined(_POSIX_TIMERS) \
```

Comment:
Suggestion was even `(defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0)`

Reply:
Just in case for reference. Without _POSIX_TIMERS > 0 another error would have been triggered on MacOS X, undeclared warning and ultimately linking error.